tests/MongoDB.Driver.Legacy.Tests/MongoCollectionTests.cs

```diff
@@ -185,23 +185,65 @@ public void TestAggregateMaxTime()
             }
         }
 
-        [Fact]
-        public void TestAggregateOutputToCollection()
+        [SkippableTheory]
+        [ParameterAttributeData]
+        public void TestAggregateOutputToCollection(
+            [Values("$out", "$merge")] string lastStageName,
+            [Values(false, true)] bool usingDifferentOutputDatabase)
         {
+            RequireServer.Check();
```

Comment:
do we use this env variable `SKIPTESTSTHATREQUIRESERVER` at all?

Reply:
Not in Evergreen. I don't use it locally either.

But we still start every integration test with `RequireServer.Check()`.

Reply:
it sounds like something that we need to change, since no reason to do it  :) We can do it later